libs/server/Storage/Session/ObjectStore/SetOps.cs

```diff
@@ -363,6 +363,49 @@ public unsafe GarnetStatus SetScan<TObjectContext>(ArgSlice key, long cursor, st
 
         }
 
+        /// <summary>
+        /// Moves a member from a source set to a destination set.
+        /// If the move was performed, this command returns 1.
+        /// If the member was not found in the source set, or if no operation was performed, this command returns 0.
+        /// </summary>
+        /// <typeparam name="TObjectContext"></typeparam>
+        /// <param name="sourceKey"></param>
+        /// <param name="destinationKey"></param>
+        /// <param name="member"></param>
+        /// <param name="smoveResult"></param>
+        /// <param name="objectStoreContext"></param>
+        internal unsafe GarnetStatus SetMove<TObjectContext>(ArgSlice sourceKey, ArgSlice destinationKey, ArgSlice member, out int smoveResult, ref TObjectContext objectStoreContext)
+            where TObjectContext : ITsavoriteContext<byte[], IGarnetObject, SpanByte, GarnetObjectStoreOutput, long>
+        {
+            smoveResult = 0;
+
+            if (sourceKey.Length == 0 || destinationKey.Length == 0 || member.Length == 0)
```

Comment:
No need for member.Length == 0, you can have a string of length 0 as a set member...